p5-travel-app/src/client/js/weatherApi.js

```diff
@@ -0,0 +1,22 @@
+const weatherKey = "b3432a956d7b45a18ba374d67c837a2e";
```

Comment:
not in code, save in env

Reply:
done in [4b6e967](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/4b6e96742131ae1774f8acaffc88a20f0b7bf430)